components/03-organisms/site-header/site-header.twig

```diff
@@ -27,7 +27,7 @@
   <div {{ bem('inner', [], site_header__base_class) }}>
     <div {{ bem('mobile-header', [], site_header__base_class) }}>
       {# Yale Branding #}
-      <a href="https://www.yale.edu/" {{ bem('site-branding', ['mobile'], site_header__base_class) }}>
+      <a href="/" {{ bem('site-branding', ['mobile'], site_header__base_class) }}>
```

Comment:
I think this should be a link component? That or maybe add a TODO here to update this to a future `branding` component.